NOAA-FIMS/FIMS
# Remove test data from fims deterministic folder

# What is the feature?


* Remove the tests/integration/inputs/FIMS-deterministic folder and its file contents
* Enable the automatic execution of the model comparison project to generate test data when running FIMS tests

# How have you implemented the solution?

- Delete the `tests/integration/inputs/FIMS-deterministic` folder and include the test data folder in the `.gitignore` file.
- Modify the `setup_gtest()` function to rename test data folder.
- Add code into `CMakeLists.txt` to execute `setup_gtest()` so that developers can have the test data readily available for C++ integration tests before running GoogleTest locally.
- Add GITHUB_PAT into `run-googletest.yml` to make sure the gha passes.

# Does the PR impact any other area of the project?

NA

# How to test this change


The GitHub Actions workflows that failed in this pull request are expected to function correctly once all changes are merged into the main branch (see [GHA workflow](https://github.com/NOAA-FIMS/FIMS/actions/runs/6565637536) that has passed during the testing of the workflow using a specific FIMS installation). This will include running `run-googletest.yml` after updating FIMS to the latest version with the revised `setup_gtest()` function.

# Developer pre-PR checklist
